Microsoft.Toolkit.Uwp.UI.Media/Extensions/System.Threading.Tasks/AsyncMutex.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Extensions.System.Threading.Tasks
```

Comment:
```suggestion
namespace Microsoft.Toolkit.Uwp.UI.Media.Extensions
```
?

Reply:
Ah, that's just ReSharper betraying me during a refactor 🤣

Reply:
Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3112/commits/5b3caef9d90e5a7f33d1ca941aa8b67c5c493d18.